todo/src/components/List.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { connect } from "react-redux";
+import { addTodo, toggleCompleted } from "../actions/actions";
+
+class List extends React.Component {
+  state = {
+    newTask: ""
+  };
+
+  handleChange = e => {
+    this.setState({ newTask: e.target.value });
+  };
+
+  addTask = e => {
+    e.preventDefault();
+    this.props.addTodo(this.state.newTask);
+  };
+
+  clearForm = e => {
+    e.preventDefault();
+    this.setState({ newTask: "" });
+  };
+
+  comboFunction = e => {
+    this.addTask(e);
+    this.clearForm(e);
+  };
```

Comment:
This is a good idea, but not really necessary. It's pretty common to clear a form or input on submit. So you can combine both the addTodo and setState back to "" into a single addTask function.

Reply:
Noted